src/resources/app-image/AppRun

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+SELF_DIR="$(dirname "$(readlink -f "$0")")"
+LIBS_PATH="$SELF_DIR"
+
+if [ -z "$LD_LIBRARY_PATH" ]; then
+  LD_LIBRARY_PATH="$LIBS_PATH"
+else
+  LD_LIBRARY_PATH="$LIBS_PATH:$LD_LIBRARY_PATH"
+fi
+
+export LD_LIBRARY_PATH
```

Comment:
The rpath is currently being set to `$ORIGIN` which should allow libraries to be loaded from next to the executable, why was `LD_LIBRARY_PATH` also needed?

Reply:
Oh you are right, that works. Thanks.